src/mpyl/reporting/targets/github.py

```diff
@@ -76,13 +76,13 @@ def __init__(
         compose_function: Callable[
             [RunResult, Optional[dict]], str
         ] = compose_message_body,
-        update_stategy: GithubUpdateStategy = GithubUpdateStategy.BODY,
+        update_strategy: GithubUpdateStrategy = GithubUpdateStrategy.BODY,
     ):
         self._raw_config = config
         self._config = GithubConfig.from_config(config)
         self.git_repository = Repository(RepoConfig.from_config(config))
         self.compose_function = compose_function
-        self.update_strategy: GithubUpdateStategy = update_stategy
+        self.update_strategy: GithubUpdateStrategy = update_strategy
         self.body_separator = "----"
```

Comment:
Unrelated, but would be good to do it here. It still happens that if the description contains the string `----` it attaches the mpyl part after it (meaning a good part of the description is now lost). Can we think of a smarter way to detect whether an mpyl run has already occurred and the description was already updated once?

Reply:
Like checking for the substring
```
----
### 📕
```